modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,251 @@
+The below example selects both the version and the xref:planning-and-tuning/runtimes/concepts.adoc[runtime] of Cypher for the same query:
+
+.Combining Cypher version selection with other query options
+[source, cypher]
+----
+CYPHER 5 runtime=parallel
+MATCH (n:Person)
+RETURN n.name
+----
+
+[[select-default-cypher-version]]
+== Select the default Cypher version when creating a database
+
+Databases created on Neo4j 2025.03 or later will have Cypher 25 as their default language.
+This is true for the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/#manage-database-systems[system, standard, and composite] Neo4j databases.
+
+To select a different default Cypher version for a database, add `DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/create-databases/[`CREATE DATABASE`] statement.
+This can be done on any Neo4j database.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DEFAULT LANGUAGE` privilege.
+
+Selecting `CYPHER 5` as the default database language ensures that every query run on that database uses the language as it existed at the time of the Neo4j 5.26 long-term support release.
+Any changes introduced in Neo4j 2025.03 or later will not affect the semantics of the query.
+
+.Create a database with Cypher 5 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_database DEFAULT LANGUAGE CYPHER 5
+----
+
+Selecting `CYPHER 25` ensures that the query will be executed using the language as it exists in the version of Neo4j that the database is currently running, provided it is on Neo4j 2025.03 or later.
+
+.Create a database with Cypher 25 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_new_database DEFAULT LANGUAGE CYPHER 25
+----
+
+[[alter-default-cypher-version]]
+== Alter the default Cypher version for a database
+
+To alter the default Cypher version on a database, add `SET DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/alter-databases/[`ALTER DATABASE`] command.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DEFAULT LANGUAGE` privilege.
+
+.Set Cypher 5 as the default language on an existing database
+[source, cypher]
+----
+ALTER DATABASE my_new_database SET DEFAULT LANGUAGE CYPHER 5
+----
+
+.Set Cypher 25 as the default language on an existing database
+[source, cypher]
+----
+ALTER DATABASE my_database SET DEFAULT LANGUAGE CYPHER 25
+----
+
+[[show-cypher-versions]]
+== Show the Cypher version of databases
+
+The Cypher version of a database can be seen using the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/listing-databases/[`SHOW DATABASES`] command as follows:
```

Comment:
would it be nice to mention that the `defaultLanguage` isn't one of the columns returned by default but does require the yield?